cartographer/mapping/internal/3d/rotation_parameterization.h

```diff
@@ -14,8 +14,8 @@
  * limitations under the License.
  */
 
-#ifndef CARTOGRAPHER_MAPPING_3D_ROTATION_PARAMETERIZATION_H_
-#define CARTOGRAPHER_MAPPING_3D_ROTATION_PARAMETERIZATION_H_
+#ifndef CARTOGRAPHER_MAPPING_INTERNAL_3D_ROTATION_PARAMETERIZATION_H_
```

Comment:
mapping internal 3d or mapping 3d internal? (I don't care as long it's consistent.)

Reply:
It is always MAPPING_INTERNAL